NEWS.md

```diff
@@ -1,3 +1,6 @@
+# etn 2.3.1
+* etn now uses a more direct dependency on `{etnservice}` for queries to a local database connection. Queries via the API and on the Lifewatch RStudio Server will now always return the same results.
```

Comment:
Please integrate the text in the section below: maybe it's only necessary that API and R Studio will return the same results.